apps/06Checkers.js

```diff
@@ -50,13 +59,35 @@ function Board() {
     }
 
     // Your code here
+    this.createCheckers = function () {
+
+        var whitePositions = [[0, 1], [0, 3], [0, 5], [0, 7],
+[1, 0], [1, 2], [1, 4], [1, 6], [2, 1], [2, 3], [2, 5], [2, 7]];
+
+        var blackPositions = [[5, 0], [5, 2], [5, 4], [5, 6],
+[6, 1], [6, 3], [6, 5], [6, 7], [7, 0], [7, 2], [7, 4], [7, 6]];
+        for (var i = 0; i < whitePositions.length; i++) {
+            var whiteCoordinates = whitePositions[i];
+            this.grid[whiteCoordinates[0][whiteCoordinates[1]]] = new Checker("white")
```

Comment:
There should be one more step here after placing the checker on the board. The next step was to add the checker to array that is keeping track of checkers in play:

`
this.checkers.push(new Checker("white"));
`